docs/fundamentals/networking/telemetry/tracing.md

```diff
@@ -0,0 +1,169 @@
+
+    ```dotnetcli
+    dotnet new aspire-starter-9 --output ConnectionTracingDemo
+    ```
+
+    Or in Visual Studio:
+
+    ![Create a .NET Aspire 9 Starter App in Visual Studio](media/aspire-starter.png)
+
+1. Open `Extensions.cs` in the `ServiceDefaults` project, and edit the `ConfigureOpenTelemetry` method adding the ActivitySources for connection in the tracing configuration callback:
+
+    :::code language="csharp" source="snippets/tracing/ConnectionTracingDemo.ServiceDefaults/Extensions.cs" id="snippet_ConnectionTracing":::
+
+1. Start the solution. This should open the [.NET Aspire Dashboard](/dotnet/aspire/fundamentals/dashboard/overview).
+
+1. Navigate to the Weather page of the `webfrontend` app to generate an `HttpClient` request towards `apiservice`.
+
+1. Return to the Dashboard and navigate to the **Traces** page. Open the `webfrontend: GET /weather` trace.
+
+[![HttpClient Spans in Aspire Dashboard](media/aspire-httpclient-get-thumb.png)](media/aspire-httpclient-get.png#lightbox)
+
+When http requests are made with the connection instrumentation enabled, you should see the following changes to the client request spans:
+
+- If a connection needs to be established, or waiting for a connection from the connection pool, then an additional [`HTTP wait_for_connection`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#http-client-request-wait-for-connection-experimental) span will be shown which represents the delay for waiting for a connection to be made. This helps to understand delays between the `HttpClient` request being made in code, and when the processing of the request actually starts. In the picture above:
+  - The selected span is the HttpClient request.
+  - The one below it's the delay waiting for a connection to be established.
+  - The last span in yellow is from the destination processing the request.
+- The HttpClient span will have a link to the [`HTTP connection_setup`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#http-connection-setup-experimental) span which represents the activity to create the http connection used by the request.
+
+[![Connection setup spans in Aspire Dashboard](media/aspire-connection_setup-thumb.png)](media/aspire-connection_setup.png#lightbox)
+
+As mentioned [in the note above](#experimental-connection-tracing), the [`HTTP connection_setup`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#http-connection-setup-experimental) span is a separate span with its own `TraceId` as its lifetime is independent from each individual client request. This span will typically have child spans [`DNS lookup`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#dns-lookup-experimental), (TCP) [`socket connect`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#socket-connect-experimental) and [`TLS client handshake`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#tls-handshake-experimental).
```

Comment:
```suggestion
As mentioned [in the previous note](#experimental-connection-tracing), the [`HTTP connection_setup`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#http-connection-setup-experimental) span is a separate span with its own `TraceId`, as its lifetime is independent from each individual client request. This span typically has child spans [`DNS lookup`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#dns-lookup-experimental), (TCP) [`socket connect`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#socket-connect-experimental), and [`TLS client handshake`](../../../core/diagnostics/distributed-tracing-builtin-activities.md#tls-handshake-experimental).
```

Reply:
Directional words like "above" are discouraged since they are useless for people using screenreaders.
https://learn.microsoft.com/en-us/style-guide/accessibility/writing-all-abilities#use-content-structure-and-location-to-communicate